config/cable.yml

```diff
@@ -5,6 +5,4 @@ test:
   adapter: async
 
 production:
-  adapter: redis
-  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
-  channel_prefix: asana_production
+  adapter: postgresql
```

Comment:
why postgres?

Reply:
The Redis adapter requires users to provide a URL pointing to the Redis server. Additionally, a channel_prefix may be provided to avoid channel name collisions when using the same Redis server for multiple applications.
The PostgreSQL adapter uses Active Record's connection pool, and thus the application's config/database.yml database configuration, for its connection.
а оскільки у нас не було редісу, а був постгрес то змінили на нього

Reply:
ставте редіс
він має бути
бо бекграунд таски є

і прав конфлікти і ще раз на ревью